Game/Source/ThirdPersonShooter/BigGym/ActorCreatorBase.cpp

```diff
@@ -0,0 +1,84 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#include "ActorCreatorBase.h"
+
+#include "Engine/World.h"
+#include "Net/UnrealNetwork.h"
+
+
+// Sets default values
+AActorCreatorBase::AActorCreatorBase()
+{
+ 	// Set this actor to call Tick() every frame.  You can turn this off to improve performance if you don't need it.
+	PrimaryActorTick.bCanEverTick = true;
```

Comment:
Don't have to do it in this PR, but could disable tick and remove the Tick and BeginPlay methods below.